src/app/static/src/app/apiService.ts

```diff
@@ -120,7 +120,12 @@ export class APIService<S extends string, E extends string> implements API<S, E>
             return
         }
 
-        let failure = e.response && e.response.data;
+        if (e.response && e.response.status == 401) {
+            window.location.assign("/login")
+        }
+
+        const failure = e.response && e.response.data;
+        /*
```

Comment:
You're right, you shouldn't need this block any more.

Reply:
removed